### DISCUSSION TITLE
Looking better docs on `scoop create` so I can add new programs

### DISCUSSION THREAD

**Discussion by didymus** [ScoopInstaller/Scoop#4485] [Help and Questions] [answered]
So far really liking Scoop. Only "complaint" I have is that it keeps getting compared to `brew`, which I use a lot, but there really isn't anywhere near the number of programs on it out-of-the-box. So, if I am understanding correctly, in order to add new programs that can be installed/managed by `scoop` I need to `create` what are called manifests. I've looked at a bunch of examples from `main` and it seems straightforward enough, but I am not sure what is "necessary" to include and what isn't so that the manifest will continue to update the software in the future. The examples are helpful, but I really could use something more explaining what I *need* to provide and also what I *should* provide. I've searched around looking for more of a tutorial-type write-up that creates a manifest, but I haven't found much. The best information I found is here on Scoop's GitHub. 

I have another related question. Once I figure out how to write good manifests, what would be the best way to share them so that others don't have to write the same ones all over? I've used GitHub before, but it was always with my own repos, so I'm not really sure how you collab with another project. I don't need anyone to just explain it to me, but I do need to know what would be the best search terms to use to find the answers I'm seeking. 

Appreciate any help.

**Comment by rashil2000** [comment, marked as answer]
Hey @didymus, have you checked out the Scoop docs (a mirror of the repo's wiki) - https://scoop-docs.now.sh/docs/concepts/App-Manifest-Autoupdate.html ?
The docs/wiki are extremely detailed, and explain the manifest concepts very nicely (be sure to read everything linked in the sidebar).

There's one [Main](https://github.com/ScoopInstaller/Main) bucket for Scoop, to which you can send a PR to add your manifest to. The Main bucket has some [Criteria](https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket) though, which needs to be satisfied. If the criteria isn't satisfied, you can contribute your manifest to the [Extras](https://github.com/ScoopInstaller/Extras) bucket instead.

**Comment by diklios5768** [comment]
> So far really liking Scoop. Only "complaint" I have is that it keeps getting compared to `brew`, which I use a lot, but there really isn't anywhere near the number of programs on it out-of-the-box. So, if I am understanding correctly, in order to add new programs that can be installed/managed by `scoop` I need to `create` what are called manifests. I've looked at a bunch of examples from `main` and it seems straightforward enough, but I am not sure what is "necessary" to include and what isn't so that the manifest will continue to update the software in the future. The examples are helpful, but I really could use something more explaining what I _need_ to provide and also what I _should_ provide. I've searched around looking for more of a tutorial-type write-up that creates a manifest, but I haven't found much. The best information I found is here on Scoop's GitHub.
> 
> I have another related question. Once I figure out how to write good manifests, what would be the best way to share them so that others don't have to write the same ones all over? I've used GitHub before, but it was always with my own repos, so I'm not really sure how you collab with another project. I don't need anyone to just explain it to me, but I do need to know what would be the best search terms to use to find the answers I'm seeking.
> 
> Appreciate any help.

You can use this to search apps you want: https://scoopsearch.github.io
And other ways can see in: https://github.com/rasa/scoop-directory
